src/app/java/changelog/layout.tsx

```diff
@@ -93,34 +89,59 @@ export default async function ChangelogLayout({
   );
 }
 
+export interface VersionEntry {
+  name: string;
+  url: string;
+  title: string;
+  description: string;
+  image: { title: string; url: string; };
+}
+
+export interface ReleaseVersionEntry extends VersionEntry {
+  nonReleaseVersions: VersionEntry[];
+  isLatest: boolean;
+}
```

Comment:
I'm not fond of exporting types from layouts/pages, I feel like `VersionEntry` would be better off being defined in `version-link.tsx` and `ReleaseVersionEntry` is better off being defined in `release-version-link-set.tsx`